profile/README.md

```diff
@@ -0,0 +1,21 @@
+# CROCODILE: Python interfaces for regional ocean modeling and data assimilation
+
+### CROCODILE is an NSF-funded cyberinfrastructure collaboration between the NSF National Center for Atmospheric Research ([NSF NCAR](https://ncar.ucar.edu/)) and the Woods Hole Oceanographic Institution ([WHOI](https://www.whoi.edu/)).
+
+### Our goal is to build a community of practice around regional ocean modeling for studying ocean physics, the carbon cycle, climate change and intervention, and more, by creating and curating a common set of open-source tools.
+
+More specifically: We want YOU to be up and running a regional ocean model in CESM, with all the necessary forcing files, boundary conditions, etc., for fairly arbitrary domains and model resolutions, including data assimilation capabilities, in a matter of hours rather than weeks!
+
+## CROCODILE combines
+- The **[MOM6](https://mom6.readthedocs.io/)** ocean model
+- **[DART](https://docs.dart.ucar.edu/en/latest/)** ensemble data assimilation
+- **[MARBL](https://marbl.readthedocs.io/)** ocean biogeochemistry
+- Coupling infrastructure from the **[Community Earth System Model (CESM)](https://www.cesm.ucar.edu/)**
+- Python infrastructure from **[regional-mom6](https://github.com/COSIMA/regional-mom6)**
+
+## Interested in contributing? Questions?
+Join us on [Slack!](https://crocodile-org.slack.com/)
```

Comment:
I think it would be good to let people know they can take a look at (most) of the code as it is being developed, and contribute.

```suggestion
- Join us on [Slack!](https://crocodile-org.slack.com/)
- Checkout our open source code being developed at [CROCODILE-CESM](https://github.com/orgs/CROCODILE-CESM/repositories)
```